docs/higher-order/tca.ipynb

```diff
@@ -0,0 +1,1448 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # it can be seen, there is a large gap between the 9th and 10th stim times
</pre><p>Move the comment to a markdown cell instead of within the code section.</p><p>Generally, I suggest to restrict inline comments to be used to mostly understand the code</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/307/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>